frontend/integration-tests/views/kubevirt/wizard.view.ts

```diff
@@ -77,3 +77,8 @@ export const cloneVmWarning = cloneVmDialog.$('.alert-warning');
 
 // VMs List view
 export const firstRowVMStatus = $('div.co-m-row:first-child > div:first-child > div:nth-child(3)');
+
+// VM wizard network
+export const dropdownOptionID = (tableType, rowNumber) => `#${tableType}-edit-${rowNumber}-row`;
+export const dropdownOption = (tableType, rowNumber) => $(dropdownOptionID(tableType, rowNumber));
+export const editableTableCell = $('.kubevirt-editable-table').$$('td');
```

Comment:
editableTableCell is maybe not necessary if `activateTableRow` is used.

Reply:
`activateTableRow` does not return an element, I need an element to do do some assertion in #L52.